test/unittests/iotools/utils/test_data_handling.py

```diff
@@ -0,0 +1,20 @@
+def test_vox_to_world_space():
+    """Test function `vox_to_world_space_method_1`.
+
+    .. note::
+        We want to verify, that the function does run with it's normal parameters
+        and that it returns what we expect it to return."""
+
+    import numpy as np
+    from nibabel.nifti1 import Nifti1Image
+
+    from clinica.iotools.utils.data_handling import vox_to_world_space_method_1
+
+    vol = np.ndarray(shape=(5, 5, 2, 100))
```

Comment:
After looking at the code in Clinica, I don't think this data is testing our use case correctly.
If you create a `Nifti1Image` like this, its header will have an `sform_code` larger than zero and `method_3` will be used instead:

https://github.com/aramis-lab/clinica/blob/cf5775924179a6e72ca340f33caeb2faf881dd35/clinica/iotools/utils/data_handling.py#L1206-L1209

I think we should test in a way similar to this instead:

```python
from nibabel.nifti1 import Nifti1Header

head = Nifti1Header()
assert not head["qform_code"] > 0
assert head["sform_code"] == 0
assert np.all(head["pixdim"] == 1)
center_coordinates = np.array([.5] * 3)
assert_array_equal(
    vox_to_world_space_method_1(coordinates_vol=center_coordinates, header=head),
    center_coordinates
)
head["pixdim"][1] = 2
assert_array_equal(
    vox_to_world_space_method_1(coordinates_vol=center_coordinates, header=head),
    np.array([1. , 0.5, 0.5])
)
```

WDYT?

Reply:
It looks better than what i'd done. And it works so it is fine by me ! I'll implement it that way :)